docs/src/main/asciidoc/logging.adoc

```diff
@@ -101,115 +152,134 @@ class SimpleBean {
    }
 }
 ----
-<1> The FQCN of the declaring class is used as a logger name, i.e. `org.jboss.logging.Logger.getLogger(SimpleBean.class)` will be used.
-<2> In this case, the name _foo_ is used as a logger name, i.e. `org.jboss.logging.Logger.getLogger("foo")` will be used.
+<1> The FQCN of the declaring class is used as a logger name, for example, `org.jboss.logging.Logger.getLogger(SimpleBean.class)` will be used.
+<2> In this case, the name _foo_ is used as a logger name, for example,`org.jboss.logging.Logger.getLogger("foo")` will be used.
 
-NOTE: The logger instances are cached internally. Therefore, a logger injected e.g. into a `@RequestScoped` bean is shared for all bean instances to avoid possible performance penalty associated with logger instantiation.
+NOTE: The logger instances are cached internally. Therefore, when a logger is injected, for example, into a `@RequestScoped` bean, it is shared for all bean instances to avoid possible performance penalties associated with logger instantiation.
 
-=== What about Apache Log4j ?
 
-link:https://logging.apache.org/log4j/2.x/[Log4j] is a logging implementation: it contains a logging backend and a logging facade.
-Quarkus uses the JBoss Log Manager backend, so you will need to include the `log4j2-jboss-logmanager` library to route Log4j logs to JBoss Log Manager.
+== Logging levels
 
-[source,xml,role="primary asciidoc-tabs-target-sync-cli asciidoc-tabs-target-sync-maven"]
-.pom.xml
-----
-<dependency>
-    <groupId>org.jboss.logmanager</groupId>
-    <artifactId>log4j2-jboss-logmanager</artifactId> <1>
-</dependency>
-----
-<1> This is the library needed for Log4j version 2; if you use the legacy Log4j version 1 you need to use `log4j-jboss-logmanager` instead.
+.The mapping between the levels
+[options="header",cols="^33%,^33%,^33%"]
+|===
+|The numerical level value |JBoss Logging level |`java.util.logging` (JUL) level
 
-[source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
-.build.gradle
-----
-implementation("org.jboss.logmanager:log4j2-jboss-logmanager") <1>
-----
-<1> This is the library needed for Log4j version 2; if you use the legacy Log4j version 1 you need to use `log4j-jboss-logmanager` instead.
+|1100 |FATAL | Not applicable
 
-You can then use the Log4j API inside your application.
+|1000 |ERROR |SEVERE
 
-WARNING: Do not include any Log4j dependencies. The `log4j2-jboss-logmanager` library includes what's needed to use Log4j as a logging facade.
+|900 |WARN |WARNING
 
-== Logging levels
+|800 |INFO |INFO
+
+|700 | Not applicable |CONFIG
 
-These are the log levels used by Quarkus:
+|500 |DEBUG |FINE
+
+|400 |TRACE |FINER
+
+|300 | Not applicable |FINEST
+|===
+* The table does not mention numerically undefined Log4j and SLF4j logging levels because they all map one-to-one with the logging levels JBoss Logging provides of the same name.
```

Comment:
I'd probably just drop this note altogether; I don't think it answers any question that a user might have.